spec/jobs/fast_track/disability_compensation_job_spec.rb

```diff
@@ -130,8 +130,8 @@
       it 'raises an error' do
         Sidekiq::Testing.inline! do
           expect do
-            subject.new.perform_async(submission_without_account_or_edpid.id, user_full_name)
```

Comment:
I saw weird failures when using `subject.new`, so I used `FastTrack::DisabilityCompensationJob` similar to other tests.

Reply:
Curious why `subject.new` was used here. Seems using `subject` is fine in other places because it calls a Sidekiq class method `subject.perform_async`?

Reply:
maybe it raised an error and then passed because the test invokes `subject.new` (rather than for the intended simulated error scenario)

Reply:
Curious if `subject.perform_async` works too but I don't really care either way

Reply:
I played around with it and don't understand why some variations work and others don't (in the back of my mind, I blaming `spring`). Regardless, it would be nice to have all the tests call `DisabilityCompensationJob` the same way.